records_mover/db/redshift/loader.py

```diff
@@ -68,7 +68,7 @@ def load(self,
                                session_token=aws_creds.token, manifest=True,
                                region=directory.loc.region,  # type: ignore
                                empty_as_null=True,
-                               **redshift_options)
+                               **redshift_options)  # type: ignore
```

Comment:
I created some initial typestubs for sqlalchemy_redshift to improve our coverage here.